### ISSUE TITLE
Trouble with installation

### ISSUE THREAD

**Issue by AlexanderPico** [PathwayCommons/grounding-search#123]
Attempting the docker route on Mac OSX 10.15. I have Docker 20.10.8 and Docker Compose version v2.0.0-rc.1 installed. I'm seeing:
```
curl: (7) Failed to connect to index port 9200: Connection refused
```

Attempting npm route, I'm seeing:
```
 http://localhost:9200/groundingsearch => connect ECONNREFUSED 127.0.0.1:9200
```

I suspect these are pointing to the same issue, but I haven't figured out what it is yet. Any suggestions?

**Comment by jvwong** [comment]
The master branch has the most up-to-date instructions (below). The `joss` branch README.md looks out of date.

<img width="894" alt="Screen Shot 2021-10-09 at 10 33 34 AM" src="https://user-images.githubusercontent.com/4706307/136662343-65a34d85-a0e1-45f5-846e-0114edf8a3ab.png">

**Comment by maxkfranz** [comment]
Yes, the `joss` branch was just for work in progress on the md and bib files.  You can look at the default branch (master) for the latest information.

I’d suggest using Docker, if possible.  If, instead, you install things like Elasticsearch manually, you may need to futz with particular versions of Java installed to the system (and ES tends to depend on old versions of Java).

If you run into issues with Docker, feel free to reach out here.  I’m away next week, but Jeff can answer Docker questions.

If you’d just like to test the API conveniently, you can see both the docs (and run queries) here: https://grounding.baderlab.org

**Comment by AlexanderPico** [comment]
Right. I followed the `master` branch instructions. I was not referring to the `joss` branch. I have Docker 20.10.8 and Docker Compose version v2.0.0-rc.1 installed. I ran `docker-compose up` and I get this error:

```
curl: (7) Failed to connect to index port 9200: Connection refused
```

**Comment by AlexanderPico** [comment]
That is ineed what I'm seeing. I waited ~5 minutes the first time. I'm trying again and am 25 minutes in and I'm still seeing this over and over.  How long is a "few minutes" :)

```
grounding    | Elasticsearch is unavailable - sleeping
grounding    |   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
grounding    |                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: index
grounding    | Elasticsearch is unavailable - sleeping
index        | OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
grounding    |   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
grounding    |                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed to connect to index port 9200: Connection refused
grounding    | Elasticsearch is unavailable - sleeping
```

**Comment by AlexanderPico** [comment]
Great. I've got 32GB, but indeed only 2GB were allocated to Docker. I changed that to 16GB and it starting loading immediatedly and was finished in ~4 minutes.  Thanks!
